contrib/completion/bash/sdk

```diff
@@ -73,12 +71,10 @@ __sdkman_complete_candidate_version() {
 				candidates+=(${version_path##*/})
 			done
 			;;
-		install)
-			local -r all_candidate_versions=$(curl --silent "${SDKMAN_CANDIDATES_API}/candidates/$candidate/${SDKMAN_PLATFORM}/versions/all")
-
+		install|i)
 			while IFS= read -d, -r version; do
 				candidates+=($version)
-			done <<< "$all_candidate_versions"
+			done <<< "$(curl --silent "${SDKMAN_CANDIDATES_API}/candidates/$candidate/${SDKMAN_PLATFORM}/versions/all")"
```

Comment:
This is fine for now until we begin caching versions too. That will happen when the upgrade command is migrated to rust.